.gitignore

```diff
@@ -19,6 +19,12 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
+# Ignore package tarball (built via "mix hex.build").
+open_api_spex-*.tar
+
+# Temporary files for e.g. tests.
```

Comment:
Necessary for this project?

Reply:
> Necessary for this project?

It was generated by default in `.gitignore` through  `mix new foobar` in the latest Elixir version. Not sure it's necessary, but good to put the default in.